src/audio/module_adapter/module/volume/ipc3/volume.c

```diff
@@ -71,7 +71,7 @@ DECLARE_TR_CTX(volume_tr, SOF_UUID(volume_uuid), LOG_LEVEL_INFO);
 #if CONFIG_FORMAT_S16LE
 /**
  * \brief Used to find nearest zero crossing frame for 16 bit format.
- * \param[in,out] source Source buffer.
+ * \param[in,out] Source buffer.
```

Comment:
I think the original was correct: name of the parameter, followed by a description. Same below in this file and in ipc4/volume.c

Reply:
yes, I know there is nothing wrong here, however, patch checker is reporting error for this, so have to made this change.

Reply:
git commit failure, you can't submit if not change, :(

Reply:
`git commit -n`